main.tf

```diff
@@ -43,3 +43,100 @@ module "vpc" {
 
 
 ##############################################################################
+
+
+##############################################################################
+# Create CBR prewired rules for VPC -> COS (scoped to resource group)
+##############################################################################
+module "slz_cbr_zone_vpcs" {
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-zone-module"
+  version          = "1.18.0"
+  name             = "${var.prefix}-slz-vpcs-zone"
+  zone_description = "Single zone grouping all SLZ VPCs."
+  account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+  addresses = [
+    for network in module.vpc :
+    { "type" = "vpc", value = network.vpc_crn }
+  ]
+}
+
+locals {
+
+  cos_data_merge_maps = merge(local.cos_data_map, local.cos_map)
+
+  cos_resource_group_names = distinct(flatten([
+    for key, value in local.cos_data_merge_maps : [
+      value.resource_group
+    ]
+  ]))
+
+  rule_contexts = [{
+    attributes = [
+      {
+        "name" : "endpointType",
+        "value" : "private"
+      },
+      {
+        name  = "networkZoneId"
+        value = module.slz_cbr_zone_vpcs["zone_id"]
+    }]
+  }]
+
+  target_service_details = [
+    for cos_rg_name in local.cos_resource_group_names :
+    {
+      target_service_name = "cloud-object-storage",
+      target_rg           = local.resource_groups[cos_rg_name]
+      tags                = var.tags
+    }
+  ]
+}
+
+module "vpc_to_cos_cbr_rule" {
+  count            = length(local.cos_resource_group_names)
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-rule-module"
+  version          = "1.18.0"
+  rule_description = "${var.prefix}-${local.target_service_details[count.index].target_service_name}-rg-scoped-slz-rule"
```

Comment:
Can we make the description more human readable. A description should describe what the rule is doing